frontend-cert/js-projects/dates-lab/user-stories.md

```diff
@@ -0,0 +1,12 @@
+1. You should create a variable named `currentDate` using `let` and assign it the current date and time using the `Date` object.
+1. You should create a function named `formatDateMMDDYYYY` that takes a date object as a parameter and returns the date formatted as `MM/DD/YYYY`.
+1. Inside the `formatDateMMDDYYYY` function, you should create variables named `month`, `day`, and `year` using `let`.
+1. The `month` variable should be assigned a String value of the month from the date object using the `getMonth` method and adding 1 to the result.
+1. The `day` variable should be assigned a String value of the day from the date object using the `getDate` method.
+1. The `year` variable should be the date object's full year using the `getFullYear` method.
+1. You should return a template literal string that uses the `month`, `day`, and `year` variables to format the date as `MM/DD/YYYY`.
+1. You should create a function named `formatDateLong` that takes a date object as a parameter.
+1. Inside the `formatDateLong` function, you should create a variable named `options` using `let`.
+1. Assign `options` an object with a `year` property set to `numeric`, a `month` property set to `long`, and a `day` property set to `numeric`.
+1. You should return a string that uses the `toLocaleDateString` method on the date object with the `options` object as an argument.
```

Comment:
Same thing here.
We should give campers the ability to come up with multiple valid answers like this one.

```js
function formatDateLong(date) {
  return date.toLocaleDateString(undefined, {
    year: "numeric",
    month: "long",
    day: "numeric",
  });
}
```

These user stories can be boiled down to just one that tests if they are returning a string that uses the `toLocaleDateString` method.

```suggestion
1. Your `formatDateLong` function should return a string that uses the `toLocaleDateString` method to format the date in a `Month Day, Year` format. Remember that the `toLocaleDateString` method accepts `locales` and `options` parameters. (Ex `toLocaleDateString(locales, options)`) For the locales, you can pass in `undefined` and for the `options` parameter, you can pass in an object like you learned about in the lecture videos.
```